src/components/ActionButton/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { Meta } from '@storybook/react';
+
+import { ReactComponent as Scheduled } from '@assets/scheduled.svg';
+
+import { ActionButton as ActionButtonComponent, ActionButtonProps } from '../ActionButton';
+import { COMPONENT } from './constants';
+
+export default {
+  title: COMPONENT.TITLE,
+  component: ActionButtonComponent,
+  parameters: {
+    docs: {
+      description: {
+        component: COMPONENT.DESCRIPTION,
+      },
+    },
+  },
+} as Meta<ActionButtonProps>;
+
+export function ActionButton(): JSX.Element {
+  const handleClick = () => {
+    console.log('ActionButtonClicked');
+  };
+
+  return (
+    <div>
+      <h3>Predefined icon</h3>
+      <div style={{ display: 'flex' }}>
+        <div style={{ marginRight: '20px' }}>
+          <ActionButtonComponent onClick={handleClick} title="Schedule an appointment" icon="scheduled" />
```

Comment:
maybe it is better to export ICONS constant and use it like this icon={ICONS.scheduled} ?

Reply:
I am not sure about it. Your icons incapsulated in your component, but. It is another way to make it. You can pass string, component, or IconComponet. If we are talking about generic component